jest.config.js

```diff
@@ -24,6 +24,9 @@ module.exports = {
 	preset: '@vue/cli-plugin-unit-jest/presets/no-babel',
 	// Allow tests in the src and in tests/unit folders
 	testMatch: ['<rootDir>/src/**/*.(spec|test).(ts|js)'],
+	transformIgnorePatterns: [
+		'node_modules/(?!(vue-material-design-icons|@juliushaertl)/)',
```

Comment:
this is already there.. shouldn't there be a conflict?

Reply:
I added it to multiple PRs to be able to move forward. Interesting that it neither got discarded nor produced a conflict.
I'll rebase and it should disappear.